helpers/helpers.cjs

```diff
@@ -540,3 +540,18 @@ function expandTildePath(file, os) {
   return homedir ? file.replace(/^~($|\/|\\)/, `${homedir}$1`) : file;
 }
 module.exports.expandTildePath = expandTildePath;
+
+/**
+ * Downcases a string and strips extra white spaces and punctuations
+ *
+ * @param {string} text String to transform
+ * @returns {string} Stripped and downcased string
+ */
+function stripAndDowncaseText(text) {
```

Comment:
Once this function is moved, I think it can be renamed something like "normalize". Otherwise, I'll want to remove the word "strip" for possible connotations.